services/matrix/matrix.tester.js

```diff
@@ -8,7 +8,7 @@ const t = new ServiceTester({ id: 'matrix', title: 'Matrix' })
 module.exports = t
```

Comment:
We're trying to switch to a shorthand for all the service tests. Would you mind adding the following line:

```js
const t = (module.exports = require('../create-service-tester')())
```

and and remove lines 4, 7, & 8?

Reply:
Done in https://github.com/badges/shields/pull/2527/commits/fad3adda24bb0a1397a3010e1b559971cf4db3e1